packages/toc/src/generators/notebook/index.ts

```diff
@@ -46,18 +47,23 @@ function createNotebookGenerator(
   settings?: ISettingRegistry.ISettings
 ): Registry.IGenerator<NotebookPanel> {
   let numberingH1 = true;
+  let collapsibleNotebooks = true;
   if (settings) {
     numberingH1 = settings.composite.numberingH1 as boolean;
+    collapsibleNotebooks = settings.composite.collapsibleNotebooks as boolean;
   }
   const options = new OptionsManager(widget, tracker, {
     numbering: false,
     numberingH1: numberingH1,
+    collapsibleNotebooks: collapsibleNotebooks,
```

Comment:
Is this considered an API change @jasongrout, @blink1073 ?

Should it be optional and `true`by default?